brainscore_vision/models/alexnet/model.py

```diff
@@ -1,28 +1,41 @@
 import functools
 
+import torch
 import torchvision.models
 
 from brainscore_vision.model_helpers.activations.pytorch import PytorchWrapper
 from brainscore_vision.model_helpers.activations.pytorch import load_preprocess_images
+from brainscore_vision.model_helpers.check_submission import check_models
 
 BIBTEX = """@incollection{NIPS2012_4824,
-                  title = {ImageNet Classification with Deep Convolutional Neural Networks},
-                  author = {Alex Krizhevsky and Sutskever, Ilya and Hinton, Geoffrey E},
-                  booktitle = {Advances in Neural Information Processing Systems 25},
-                  editor = {F. Pereira and C. J. C. Burges and L. Bottou and K. Q. Weinberger},
-                  pages = {1097--1105},
-                  year = {2012},
-                  publisher = {Curran Associates, Inc.},
-                  url = {http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf}
-                  }"""
+    title = {ImageNet Classification with Deep Convolutional Neural Networks},
+    author = {Alex Krizhevsky and Sutskever, Ilya and Hinton, Geoffrey E},
+    booktitle = {Advances in Neural Information Processing Systems 25},
+    editor = {F. Pereira and C. J. C. Burges and L. Bottou and K. Q. Weinberger},
+    pages = {1097--1105},
+    year = {2012},
+    publisher = {Curran Associates, Inc.},
+    url = {http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf}
+}"""
 
 LAYERS = ['features.2', 'features.5', 'features.7', 'features.9', 'features.12',
           'classifier.2', 'classifier.5']
 
 
 def get_model():
-    model = torchvision.models.alexnet(pretrained=True)
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
I think this should rather be in the wrapper/activations helper (which has to also move stimuli to gpu)

Reply:
You're correct. It already exists in the Pytorch wrapper, so no need for it here.